modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/option/SabrInterestRateParameters.java

```diff
@@ -136,55 +136,55 @@ public static SabrInterestRateParameters of(
 
   //-------------------------------------------------------------------------
   /**
-   * Return the alpha parameter for a pair of time to expiry and instrument tenor.
+   * Calculates the alpha parameter for a pair of time to expiry and instrument tenor.
    * 
    * @param expiry  the expiry
    * @param tenor  the tenor
```

Comment:
Is it worth specifying the tenor is a year fraction? Assuming it is a year fraction that is.